### DISCUSSION TITLE
How is may-minihttp so fast compared to other libraries?

### DISCUSSION THREAD

**Discussion by DanielCoder834** [TechEmpower/FrameworkBenchmarks#9093] [General]
On the benchmark test result [site](https://www.techempower.com/benchmarks/#hw=ph&test=fortune&section=data-r22), it shows that may-minihttp is the fastest web framework. I was wondering why that is the case.

**Comment by volyrique** [comment]
While I don't have an aswer to your question (I am in the process of experimenting with my own implementation), note that `may-minihttp` is (clearly) faster than other frameworks in only 2 out of 7 tests - fortunes and single query; both of them are database tests. So, your best bet is that it has optimized its communication with the database.

**Reply by DanielCoder834** [reply]
What does fortunes mean?  Also would you happen to know why ntex is so fast for queries and data updates? Also would that mean that ntex is better than may-minihttp in terms of speed regarding communication with databases?

**Reply by volyrique** [reply]
Fortunes is [one](https://github.com/TechEmpower/FrameworkBenchmarks/wiki/Project-Information-Framework-Tests-Overview#fortunes) of the defined test types. If you check the latest results [here](https://tfb-status.techempower.com/), you will notice that `ntex` is not faster than any of the other implementations, especially in the tests that you mention.

**Reply by DanielCoder834** [reply]
thank you
